<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetHodor</class>
 <widget class="QWidget" name="WidgetHodor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>642</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1025</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>904</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>904</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>904</width>
     <height>2</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 144, 144);
border:0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>60</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>60</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>301</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>301</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QGroupBox {
	border-width:1px;
	border-style:solid;
	border-color:lightGray;
	margin-top: 0.5ex;
	border-radius:10px;
color: rgb(255, 255, 255);
font: 28px &quot;.Helvetica Neue DeskInterface&quot;;
}
QGroupBox::title {
   subcontrol-origin: margin;
   subcontrol-position: top left;
   left:10px;
   margin-left: 0px;
   padding:0 1px;
}</string>
        </property>
        <property name="title">
         <string>设备信息</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ADB:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_adb">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>DID:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_did">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>KEY:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_key">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>MAC:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_mac">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>SN:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_sn">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>602</width>
          <height>301</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>301</width>
          <height>602</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">

QTableWidget{
	color: rgb(144, 144, 144);
	gridline-color:rgb(144, 144, 144);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
	color: rgb(255, 255, 255);
	background-color: transparent; 
	border-radius:10px
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>温湿度</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>光线</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="3">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>重力</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="1">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>按键</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="3">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>触摸</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="1">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>Wi-Fi</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="3">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>TVOC&amp;CO2</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="3" column="1">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="3" column="2">
         <property name="text">
          <string>电池</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="3" column="3">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>PM2.5</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="4" column="1">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="4" column="2">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="4" column="3">
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_sync">
        <property name="minimumSize">
         <size>
          <width>288</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>288</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-image: url(:/images/button/button1.png);
	color: rgb(66, 66, 66);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}
QPushButton:pressed{
border-image: url(:/images/button/button1_press.png);
}
QPushButton:disabled{
border-image: url(:/images/button/button1_press.png);
}</string>
        </property>
        <property name="text">
         <string>上报测试结果</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_result">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QWidget" name="page_normal"/>
        <widget class="QWidget" name="page_success">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 230, 118);</string>
            </property>
            <property name="text">
             <string>成功</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_failed">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(244, 67, 54);</string>
            </property>
            <property name="text">
             <string>失败</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
